scripts/apps/authoring/authoring/ValidateCharactersConnected.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {EditorState} from 'draft-js';
+import {ValidateCharacters} from './ValidateCharacters';
+
+interface IProps {
+    editorState: EditorState;
+    fieldId: string;
+}
+
+export class ValidateCharactersComponent extends React.PureComponent<IProps> {
+    render() {
+        const text = this.props.editorState.getCurrentContent().getPlainText();
+
+        return (
+            <span className="disallowed-char-error" style={{float: 'none', margin: 0}}>
```

Comment:
This combination of a className and inline styles repeats. I think it would be better if we have it as another class to be reused

Reply:
this file will be removed along with angular based authoring and the issue you pointed out is quite minor, so I think it's not worth improving this